src/shipchain_common/views.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
Should maybe be under mixins.py instead of views.py? I'm not sure of the best practice for this. I could also even see them going respectively in permisisons.py/serializers.py

Reply:
Would it make sense to create a mixins folder, and within that put the file? Would be an easy/nice way of organizing mixins in the future.

Reply:
I wasn't sure of the best practice either.  My reasoning was since these mixins are _used_ in views, that's where they would be referenced from.  However, I'm open to whatever you like best

Reply:
:man_shrugging: